client/lib/plans/constants.js

```diff
@@ -79,10 +79,8 @@ export const plansList = {
 		getProductId: () => 1,
 		getStoreSlug: () => PLAN_FREE,
 		getPathSlug: () => 'beginner',
-		getDescription: () => i18n.translate(
-			'Get a free blog and be on your way to publishing your first post' +
-			' in less than five minutes.'
-		),
+		getDescription: () => i18n.translate( 'Get a free blog and be on your way to publishing your first post in less than five minutes.' ),
+		getTargetedDescription: () => i18n.translate( 'Get a free website and be on your way to publishing your first post in less than five minutes.' ),
```

Comment:
I think this was a bad rebase on my part.. @apeatling did you have special targeted description for the free plan? Doesn't look so from your screenshot.